modules/ethereum/src/import.rs

```diff
@@ -166,214 +166,221 @@ pub fn header_import_requires_receipts<S: Storage>(
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use crate::tests::{
-		block_i, custom_block_i, genesis, signed_header, validator, validators_addresses, InMemoryStorage,
+	use crate::mock::{
+		block_i, custom_block_i, custom_test_ext, genesis, signed_header, test_aura_config, test_validators_config,
+		validator, validators, validators_addresses, TestRuntime,
 	};
 	use crate::validators::ValidatorsSource;
-	use crate::{kovan_aura_config, kovan_validators_config};
+	use crate::{BridgeStorage, Headers, OldestUnprunedBlock};
+	use frame_support::{StorageMap, StorageValue};
 
 	#[test]
 	fn rejects_finalized_block_competitors() {
-		let mut storage = InMemoryStorage::new(genesis(), validators_addresses(3));
-		storage.finalize_headers(Some((100, Default::default())), None);
-		assert_eq!(
-			import_header(
-				&mut storage,
-				&kovan_aura_config(),
-				&kovan_validators_config(),
-				PRUNE_DEPTH,
-				None,
-				Default::default(),
-				None,
-			),
-			Err(Error::AncientHeader),
-		);
+		custom_test_ext(genesis(), validators_addresses(3)).execute_with(|| {
+			let mut storage = BridgeStorage::<TestRuntime>::new();
+			storage.finalize_headers(Some((100, Default::default())), None);
+			assert_eq!(
+				import_header(
+					&mut storage,
+					&test_aura_config(),
+					&test_validators_config(),
+					PRUNE_DEPTH,
+					None,
+					Default::default(),
```

Comment:
Does this test work because the default block number would be 0? If so, it would be nice to make that more explicit

Reply:
Logged #111